src/consumer/frame.js

```diff
@@ -53,6 +53,12 @@ class Frame extends EventEmitter {
           return Promise.resolve();
         },
 
+        unload(detail = {}) {
+          self.wrapper.setAttribute('data-status', 'unloading');
```

Comment:
is this needed?

Reply:
its to be consistent about tracking lifecycle events. This hypothetically allows specific styling on the iframe while it is between and "unloaded" and "launched" state. 

Regarding that, I'd probably call it `unloaded` instead of unloading.